ziti/cmd/edge/quickstart.go

```diff
@@ -75,8 +75,7 @@ func NewQuickStartCmd(out io.Writer, errOut io.Writer, context context.Context)
 	cmd.Flags().StringVarP(&options.Username, "username", "u", "", "Username to use when creating the Ziti Edge Controller. default: admin")
 	cmd.Flags().StringVarP(&options.Password, "password", "p", "", "Password to use for authenticating to the Ziti Edge Controller. default: admin")
 
-	cmd.Flags().BoolVar(&options.AlreadyInitialized, "already-initialized", false, "Specifies the PKI does not need to be created and the db does not need to be initialized. Recommended to be combined with --home. If --home is not specified the environment will be destroyed on shutdown! default: false")
-	cmd.Flags().StringVar(&options.Home, "home", "", "Sets the directory the environment should be installed into. Defaults to a temporary directory. If specified, the environment will not be removed on exit.")
+	cmd.Flags().StringVar(&options.Home, "home", "", "persistent state directory")
```

Comment:
--> "Sets the directory the environment will be installed into. When set, this location will not be deleted on exit"